test/utils/clients.go

```diff
@@ -19,27 +19,31 @@ limitations under the License.
 package utils
 
 import (
+	operatorVersioned "github.com/tektoncd/operator/pkg/client/clientset/versioned"
+	pipelineVersioned "github.com/tektoncd/pipeline/pkg/client/clientset/versioned"
+
```

Comment:
nit: not extra "new line" is needed here.

Reply:
fixed